physics.py

```diff
@@ -126,7 +125,7 @@ def update_sim(self, now: float, tm_diff: float) -> None:
             )
         )
 
-        self.imu_yaw.set(self.imu_yaw.get() - math.degrees(speeds.omega * tm_diff))
+        self.imu.add_yaw(math.degrees(speeds.omega * tm_diff))
```

Comment:
Isn't yaw in radians in Phoenix 6?

Reply:
![image](https://github.com/user-attachments/assets/16827873-9d21-4472-b445-f9a6a1e4cbc1)
Could be, but this is the function annotation telling me degrees

Reply:
It seems all the sim functions are in degrees

Reply:
> in rotations
> type: degrees

wtf CTRE